windows/src/global/delphi/general/Keyman.System.CanonicalLanguageCodeUtils.pas

```diff
@@ -17,8 +17,6 @@ implementation
   Keyman.System.Standards.LangTagsRegistry,
   System.SysUtils;
 
-// TODO: Make this a COM API function so we aren't shipping 3+MB of standards data multiple times
```

Comment:
This has already been done; see `IKeymanBCP47Canonicalization` (implemented by `keyman_implementation.TKeyman`)